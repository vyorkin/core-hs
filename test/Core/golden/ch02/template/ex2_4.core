main = S K K 3
