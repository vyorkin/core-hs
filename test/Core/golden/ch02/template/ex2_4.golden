1 : NSupercomb main
13 : NAp 11 12
11 : NAp 10 3
13 : NAp 11 12
10 : NAp 5 3
11 : NAp 10 3
13 : NAp 11 12
5 : NSupercomb S
10 : NAp 5 3
11 : NAp 10 3
13 : NAp 11 12
16 : NAp 14 15
14 : NAp 3 12
16 : NAp 14 15
3 : NSupercomb K
14 : NAp 3 12
16 : NAp 14 15
12 : NNum 3
Stats:
  Total number of steps: 8